zimit.py

```diff
@@ -431,14 +437,30 @@ def cleanup():
     elif crawl.returncode != 0:
         raise subprocess.CalledProcessError(crawl.returncode, cmd_args)
 
-    warc_files = list(temp_root_dir.rglob("collections/crawl-*/archive/"))[-1]
-    warc2zim_args.append(str(warc_files))
-
-    num_files = sum(1 for e in warc_files.iterdir())
+    if zimit_args.collection:
+        warc_files = temp_root_dir.joinpath(
+            f"collections/{zimit_args.collection}/archive/"
+        )
+    else:
+        warc_files = list(temp_root_dir.rglob("collections/crawl-*/archive/"))
+        if len(warc_files) == 0:
+            raise RuntimeError(
+                "Failed to find directory where WARC files have been created"
+            )
+        elif len(warc_files) > 1:
+            print("Found many WARC files directories, only last one will be used")
+            for directory in warc_files:
+                print(f"- {directory}")
+        warc_files = warc_files[-1]
```

Comment:
I don't like changing a variable's type. We use it only a couple times, maybe we can use the `[-1]` there